src/components/data_custodian_table/DataCustodianTable.js

```diff
@@ -310,16 +310,16 @@ export default function DataCustodianTable(props) {
   };
 
   const removeDataCustodian = async (selectedResearcher, researchers) => {
-    const { displayName, email, dacUserId } = selectedResearcher;
-    const searchableKey = !isNil(dacUserId) ? 'dacUserId' : 'email';
+    const { displayName, email, userId } = selectedResearcher;
+    const searchableKey = !isNil(userId) ? 'dacUserId' : 'email';
```

Comment:
```suggestion
    const searchableKey = !isNil(userId) ? 'userId' : 'email';
```
Is the logic here correct? if the userId is not Nil, we should be using it for the searchableKey, right?

Reply:
Yes, I think this logic still holds true here.